### PR TITLE
Create WG Reliability

### PR DESCRIPTION
**Pull request by wojtek-t**
Email thread: https://groups.google.com/forum/#!topic/kubernetes-dev/wx2qjnPEzc4

Proposed charter: https://docs.google.com/document/d/1IZnIG26FOJXWvW5n0Yu6zL89pH4cO8NYHKMiESc2fV0/edit#

### CODE REVIEW

**Comment by wojtek-t** [comment]
@deads2k @stevekuznetsov

**Comment by johnbelamaric** [comment]
+1 for sig-arch sponsorship

**Comment by wojtek-t** [comment]
[For completenes]
+1 for SIG Scalability sposorship

File: sigs.yaml
Comment (dims) [inline sigs.yaml:2772]: `Allow users to safely use Kubernetes for managing production workloads by ensuring Kubernetes is stabie and reliable.`

reads a little bit better. wdyt?

File: wg-reliability/README.md
Comment (dims) [inline wg-reliability/README.md:11]: same as a above.

**Comment by dims** [comment]
@wojtek-t the README is a bit sparse can you please check other readme(s) like this one for example ( https://github.com/kubernetes/community/blob/master/wg-naming/README.md ) Looks like there's plenty of material in https://docs.google.com/document/d/1IZnIG26FOJXWvW5n0Yu6zL89pH4cO8NYHKMiESc2fV0/edit which can possibly help populate the README.

**Comment by stevekuznetsov** [comment]
Spoke with @spiffxp 

+1 for sig-testing sponsorship

**Comment by cblecker** [comment]
/hold
/committee steering
for review :)

**Comment by spiffxp** [comment]
/lgtm
as sig testing chair and a steering committee member

File: sigs.yaml
Comment (wojtek-t) [reply sigs.yaml:2772]: done

File: wg-reliability/README.md
Comment (wojtek-t) [reply wg-reliability/README.md]: done

**Comment by justaugustus** [comment]
/cc @justaugustus @tpepper 
(for SIG Release)

**Comment by wojtek-t** [comment]
@justinsb @neolit123 @timothysc @fabriziopandini - for ClusterLifecycle sposorship too

**Comment by neolit123** [comment]
+1, i have read the related proposals.
thanks.

**Comment by spiffxp** [comment]
/lgtm
putting the lgtm back on

File: wg-reliability/README.md
Comment (tpepper) [inline wg-reliability/README.md:11]: This one sentence feels very broad for scope.  Scope was a large part of the other discussion threads before this PR. Without asking you to go all the way into including a draft charter doc in this PR, could some of that discussion be distilled into a few more sentences or bullet points here tentatively describing the expected outline shape of "ensuring Kubernetes is stable and reliable"?

**Review by tpepper — CHANGES_REQUESTED** [review]

File: wg-reliability/README.md
Comment (tpepper) [reply wg-reliability/README.md:11]: ie: this info exists, it'll just be hard to find for a reader of the updated sigs list file once this merges and they're jumping off to this PR's current readme unless they look at the file git history and find this pr and the links in its description and go to them.

Comment (dims) [reply wg-reliability/README.md:11]: @tpepper this is the README, this WG still has to file a charter with the scope ( example https://github.com/kubernetes/community/blob/master/wg-k8s-infra/charter.md#scope )

**Comment by dims** [comment]
/lgtm

(wearing sig-arch and steering hats)

File: wg-reliability/README.md
Comment (wojtek-t) [reply wg-reliability/README.md:11]: which will be roughly what is here:
https://docs.google.com/document/d/1IZnIG26FOJXWvW5n0Yu6zL89pH4cO8NYHKMiESc2fV0/edit#

I'm not entirely sure it makes sense to put it here - if the WG will get approved I can put together a charter PR in the next day (basically migrating this doc to the markdown in this repo).

**Comment by krmayankk** [comment]
have read the proposal and +1 for prioritizing reliability.

File: wg-reliability/README.md
Comment (tpepper) [reply wg-reliability/README.md:11]: Are we expecting a fast chartering here?  Some WG's have been quite slow.  If this one gets slow, at least a minimal summary of potential scope vision and a link to the draft charter would be beneficial in the meantime for somebody arriving at the sig-list.md without connection to details.

Comment (wojtek-t) [reply wg-reliability/README.md:11]: As soon as this is approved, I will open PR with the charter (mostly what is in https://docs.google.com/document/d/1IZnIG26FOJXWvW5n0Yu6zL89pH4cO8NYHKMiESc2fV0/edit#heading=h.jrj8xt906vjp
). Given more-or-less agreement on that (number of people already looked at this and commented), I think it hopefully shouldn't be very controversial, so hopefully it should come almost immediately after WG approval.

**Review by tpepper — APPROVED** [review]
I have some questions on scope plus overlap versus https://github.com/kubernetes/sig-release/issues/966 but I'm OK with this proceeding and have assessed a generally similar view amongst those active in SIG Release.  It matters less where this work is coordinated and for how long (eg: a subproject could carry things for the long term with specific code ownership, versus a WG being time boxed and not necessarily code owning) than getting more focus and concerted effort on the topic space.

/lgtm

**Comment by wojtek-t** [comment]
> It matters less where this work is coordinated and for how long ... than getting more focus and concerted effort on the topic space.

Yeah - wrt CI Signal what is really important is agreement on goals, criteria etc. and we absolutely are not going to push that without your agreements. I agree that there is small overlap, but it actually isn't that big and I'm sure we will figure it out easily.

**Comment by justaugustus** [comment]
+1 on SIG Release sponsorship

**Comment by wojtek-t** [comment]
Thanks all!

@kubernetes/steering-committee - it seems that all sposorships are already here. Can we proceed with having the WG approved?

**Comment by dims** [comment]
/approve
/lgtm

(steering hat on)

**Comment by spiffxp** [comment]
@kubernetes/steering-committee 
We need two more steering committee members to +1 or /lgtm to reach the simple majority needed to move this forward

**Comment by nikhita** [comment]
/lgtm
/approve

(steering hat on)

**Comment by parispittman** [comment]
lgtm

**Comment by dims** [comment]
/hold cancel

we have simple majority on steering.

thanks for taking point on this @wojtek-t

**Comment by wojtek-t** [comment]
Thanks!

We're working on configuring communication channels, I will open a PR with charter etc. in the next day or so. I would like the WG to start meeting starting since the beginning of October. Stay tuned for more information in the next couple days.

**Comment by wojtek-t** [comment]
To close the loop on it - the kubernetes-wg-reliability@ mailing list has been created as well as wg-reliability slack channel.
If you're interested, please join, we will be sending further updates there.

**Pull request merged**
